docs/examples/vector_stores/WeaviateIndex_auto_retriever.ipynb

```diff
@@ -0,0 +1,719 @@
+{
```

Comment:
<p>could you avoid output embeddings since they are too long. you could just output the info from the nodes</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/run-llama/llama_index/pull/11885/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>